packages/react-native/package.json

```diff
@@ -36,6 +36,7 @@
   },
   "peerDependencies": {
     "aws-amplify": "^6.3.2",
+    "react": "^16.14.0 || ^17.0 || ^18.0",
```

Comment:
Have we validated that our supported versions of RN work with all of these React versions?

Reply:
Updated this to use `"*"`